src/collections/news/ announcing-development-of-trust-meter-technical-specification.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: layouts/single--news.njk
+title: Announcing development of Trust Meter Technical Specification
+date: 2025-04-14T18:22:00Z
+author: IDRC Team
+excerpt: IDRC would like to announce development of a new Technical Specification
+  for Standards Council of Canada (SCC) called Trust Meter.
+
+---
+IDRC would like to announce development of a new Technical Specification for Standards Council of Canada (SCC) called Trust Meter.
+
+The Trust Meter Technical Specification will support development of an inclusive and balanced data environment, beginning with persons with disabilities and extending to other marginalized groups. The Trust Meter Technical Specification outlines the requirements for notifying AI systems operators when the scenario, group or individual about whom the AI-based decision is made, is out-of-distribution relative to the training set the model is trained on. In such cases, the AI is unreliable to make these decisions.
+
+The Trust Meter project is being advanced under SCC’s AI and Data Governance (AIDG) Program. For more information about this project, please contact Vera Roberts ([vroberts@ocadu.ca](mailto:vroberts@ocadu.ca)).
+
+---
```

Comment:
Rendered page has a horizontal rule with some margin at top and no margin at bottom, so it looks a little off. I am not sure what would be the best way to address this issue, as the top margin is from the paragraph element before the hr, so maybe set a style for a div follows a hr to have the same amount of margin top? Though, I am not sure if the margin amount is the same for different elements. Might be an overwork, but a way would be to remove margin bottom from the previous element of hr and margin top of the following element, then set margin-block to hr.